### PR TITLE
fix issue #18 user by ID

### PR DESCRIPTION
**Pull request by mohitkh7**

### CODE REVIEW

File: api/modules/city/views.py [hunk 1]
```diff
@@ -18,9 +18,10 @@ def get_all_cities(request, no_of_cities=8):
     :param no_of_cities: (default count: 8)
     :return: 200 successful
     """
-    cities = City.objects.all().order_by('-id')[:no_of_cities]
```
Comment (prabhakar267) [inline api/modules/city/views.py]: Remove the unwanted changes

File: api/urls.py [hunk 2]
```diff
@@ -14,10 +14,10 @@
     url(r'^sign-in', auth_views.obtain_auth_token, name='sign-in'),
 
     # Users
+    path('get-user/<int:user_id>', user_views.get_user_by_id, name='get-user-by-id'),
     path('get-user/<str:email>', user_views.get_user, name='get-user'),
 
     # City APIs
-    path('get-all-cities', city_views.get_all_cities, name='get-all-cities'),
```
Comment (prabhakar267) [inline api/urls.py]: remove this change

**Review by prabhakar267 — CHANGES_REQUESTED** [review]
Squash all your commits into one

**Comment by mohitkh7** [comment]
I tried squashing commits, but things got messed up. Created new pull request #22

**Pull request closed without merging**